esmvaltool/diag_scripts/monitor/monitor.py

```diff
@@ -27,6 +27,11 @@
       produce multi panel plots for data with `shape_id` or `region`
       coordinates of length > 1. Supported coordinates: `time`, `shape_id`
       (optional) and `region` (optional).
+    - Diurnal cycle (plot type ``diurnal_cycle``): Generate a diurnal cycle
+      plot (timeseries like climatological from 0 to 24 hours). It will
```

Comment:
That is a data limitation. We wrote 0-24 hours as (if data are available), one could plot 0:00:00 to 23:59:59.

Reply:
> That is a data limitation. 
Does MIROC not provide a full hourly timeseries? Or some days are missing some hours and the diurnal cycle mean requires them all?

> We wrote 0-24 hours as (if data are available), one could plot 0:00:00 to 23:59:59.
Thanks, that makes sense.

Reply:
The 3-hourly CMIP6 output used in the example provides data at 1.30 (0.00-3.00), 4.30 (3.00-6.00), ..., 22.30 (21.00-24.00). CMIP6 data are converted to full hours by preprocessor `resample_hours` so that they can be compared to ERA5 data (provided at full hours). The preprocessor `distance_metric,` applied afterwards requires all datasets to have the same time coordinate.